demo/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Web Components demo</title>
+
+  <!-- ultimately should be something like https://unpkg.com/attractions@2.2.0/dist/index.js -->
+  <script src="../attractions/dist/bundle.js"></script>
+</head>
+
+<body>
+  <a-card>
+    <a-button filled="true" danger="true">
```

Comment:
is it even possible to use any props that are not boolean or string? I'm pretty sure HTML treats all props as strings

Reply:
Right, but a string is still truthy, so we should be ok here as long as we don't compare with `=== true` or similar. About other data types, that still needs investigation